TeaDiary.business/Values/ITeaGetter.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using TeaDiary.domain.Models;
+
+namespace TeaDiary.business.Values
+{
+    public interface ITeaGetter
+    {
+        IList<Tea> GetAll();
+        Tea GetByID(int id);
```

Comment:
`ID` should be `Id`

Reply:
Should it be `ID` in tea and user models as well?

Reply:
It should always be `Id`. Consider it to be a word and not an acronym.

Reply:
I hope I made it right everywhere